.changeset/four-houses-invite.md

```diff
@@ -0,0 +1,13 @@
+---
+"@logto/core": minor
```

Comment:
should this be a patch?

Reply:
should be patch, and the commit message should starts with `refactor: `?